src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/aws/AWSEC2JCloudsProvider.java

```diff
@@ -63,13 +62,13 @@
 import com.google.common.collect.Sets;
 
 import lombok.Getter;
+import lombok.extern.log4j.Log4j2;
 
 
 @Component
+@Log4j2
 public class AWSEC2JCloudsProvider extends JCloudsProvider {
 
-    private static final Logger logger = Logger.getLogger(AWSEC2JCloudsProvider.class);
-
     @Getter
     private final String type = "aws-ec2";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)